storybookjs/storybook
# UI: Show the story status in the search results

Follow-up from: https://github.com/storybookjs/storybook/pull/23342 and https://github.com/storybookjs/storybook/pull/22890

## What I did

I added the colored icon to the search-results similar to the Tree-items in the Sidebar component.
I added a story for increased coverage

## How to test

The newly added story should demonstrate the behavior.
If you want to reproduce for yourself, please check the steps to test on this PR: https://github.com/storybookjs/storybook/pull/23342

And then use the search feature in the sidebar

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers


`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
